include/manif/impl/se2/SE2_base.h

```diff
@@ -414,11 +414,12 @@ struct AssignmentEvaluatorImpl<SE2Base<Derived>>
   static void run_impl(const T& data)
   {
     using std::abs;
-    using Scalar = typename SE2Base<Derived>::Scalar;
-    MANIF_ASSERT(abs(data.template tail<2>().norm()-Scalar(1)) <
-                 Constants<Scalar>::eps_s,
-                 "SE2 assigned data not normalized !",
-                 invalid_argument);
+    MANIF_ASSERT(
+      abs(data.template tail<2>().norm()-typename SE2Base<Derived>::Scalar(1)) <
+      Constants<typename SE2Base<Derived>::Scalar>::eps_s,
+      "SE2 assigned data not normalized !",
+      invalid_argument
+    );
```

Comment:
I guess the RELEASE mode was playing tricks here regarding the using directive?

Reply:
The `MANIF_ASSERT` macro, in release, resolve to a blank (`#def MANIF_ASSERT() ((void)0)`). Because of that the local `Scalar` type was defined but not used, which was picked up as an error by the strict compilation flags used in our tests.